packages/icons/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@livechat/design-system-icons",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "build": "svgr --typescript -d ./dist ./src/material ",
```

Comment:
What do you think about putting the whole `src` folder as an input?

Reply:
that would create a "material" directory inside "dist", do we want that?